build.sh

```diff
@@ -93,8 +93,11 @@ cd ../..
 
 echo ""
 echo "Copying extra plugins..."
-cp -r plugins/* ckeditor/plugins/
-
+if [ "$(uname)" == "Darwin" ]; then
+	cp -R plugins/ ckeditor/plugins/
```

Comment:
On OSX the following two commands give equal results:

```
cp -R source/* dest/source/
cp -R source/ dest/source/
```
that was the reason why for OSX removed the superfluous asterisk (note that we're copying into a destination folder with the same name). I will add it back then.